Duplicati/Library/Backend/AzureBlob/AzureBlobBackend.cs

```diff
@@ -195,11 +201,85 @@ public string Description
         public string[] DNSName
         {
             get { return _azureBlob.DnsNames; }
-        }
+        }
+
 
         public void Test()
-        {
-            this.TestList();
+        {
+            // Test enumeration
+            var entriesEnumerator = _azureBlob.ListContainerEntries().GetEnumerator();
+            try
+            {
+                for (int i = 0; i < TEST_FILE_ENUM_COUNT && entriesEnumerator.MoveNext(); i++)
+                {
+                    var fileEntry = entriesEnumerator.Current;
+                }
+            }
+            finally
+            {
+                entriesEnumerator.Dispose();
+            }
+
+
+

```

Comment:
Can we remove redundant blank lines here (and below as well)?  We prefer not to have more than one consecutive blank line.